app/main.py

```diff
@@ -1 +1,13 @@
 # write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
```

Comment:
```suggestion
    split_command = command.split()
```

Space is a default argument in `split`